etc/kayobe/kolla.yml

```diff
@@ -216,11 +216,18 @@ stackhpc_ubuntu_focal_repos:
   - "deb {{ stackhpc_repo_ubuntu_focal_security_url }} focal-security main universe"
   - "deb {{ stackhpc_repo_ubuntu_cloud_archive_url }} focal-updates/wallaby main"
 
+# Whether to revert to the upstream mirrors in built Kolla container images.
+stackhpc_kolla_clean_up_repo_mirrors: true
+
 # Dict mapping Jinja2 block names in kolla's Docker images to their contents.
 kolla_build_blocks:
   base_header: |
     {% if kolla_base_distro == 'centos' %}
     RUN \
+    {% if stackhpc_kolla_clean_up_repo_mirrors | bool %}
+        mkdir -p /etc/yum.repos.d.backup && \
+        tar -czf /etc/yum.repos.d.backup/repos.tar.gz -C /etc/yum.repos.d . && \
```

Comment:
Can we be sure tar exists at this point?

Reply:
At this point it's just the base CentOS container image, so we haven't installed tar, but it was present when I tested this.